test/verify/check-machines-dbus

```diff
@@ -292,6 +295,60 @@ class TestMachinesDBus(machineslib.TestMachines):
         b.wait_in_text("#vm-subVmTest1-network-1-source", "test_network")
         b.wait_not_present("#vm-subVmTest1-network-1-source-tooltip")
 
+        # Remove the network interface
+        m.execute("virsh detach-interface --persistent --type network --domain subVmTest1")
+
+        # We don't get events for shut off VMs so reload the page
+        b.reload()
+        b.enter_page('/machines')
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_in_text(".cards-pf #card-pf-networks .card-pf-title-link", "Networks")
+        b.wait_in_text("#card-pf-networks .card-pf-aggregate-status-count", "2")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+        b.click("tbody tr[data-row-id=vm-subVmTest1] th") # click on the row header
+        b.click("#vm-subVmTest1-networks") # open the "Networks" subtab
+
+        # Change network type and source from the bridge NIC
+        b.wait_in_text("#vm-subVmTest1-network-1-type", "bridge")
+        b.wait_in_text("#vm-subVmTest1-network-1-source", "virbr0")
+
+        # Open the modal dialog
+        b.click("#vm-subVmTest1-network-1-edit-dialog")
+
+        b.wait_in_text("#vm-subVmTest1-network-1-select-type", "Bridge to LAN")
+        b.select_from_dropdown("#vm-subVmTest1-network-1-select-type", "Virtual network")
+        b.select_from_dropdown("#vm-subVmTest1-network-1-select-source", "test_network")
+
+        # Save the network settings
+        b.click("#vm-subVmTest1-network-1-edit-dialog-save")
+        b.wait_not_present("#vm-subVmTest1-network-1-edit-dialog-modal-window")
+
+        b.wait_in_text("#vm-subVmTest1-network-1-type", "network")
+        b.wait_in_text("#vm-subVmTest1-network-1-source", "test_network")
+
+        # bridge-utils are not available on RHEL test images
+        if m.image not in ["rhel-8-0-distropkg", "rhel-8-1"]:
```

Comment:
This needs "rhel-8-0" as well then. But please file a PR to rebuild the images with bridge-utils.

Reply:
Sure, I am on it.